project/Build.scala

```diff
@@ -1331,16 +1331,32 @@ object Build {
         val inkuireLink = s"https://github.com/VirtusLab/Inkuire/releases/download/$inkuireVersion/inkuire.js"
         val inkuireDestinationFile = (Compile / resourceManaged).value / "dotty_res" / "scripts" / "inkuire.js"
         sbt.IO.touch(inkuireDestinationFile)
-        val downloadProcess = (new java.net.URL(inkuireLink) #> inkuireDestinationFile).run()
-        val result: Future[Int] = Future(blocking(downloadProcess.exitValue()))
-        val res = try {
-          Await.result(result, duration.Duration(60, "sec"))
-        } catch {
-          case _: TimeoutException =>
-            downloadProcess.destroy()
-            throw new MessageOnlyException(s"Failed to fetch inkuire.js from $inkuireLink: Download timeout")
+
+        def tryFetch(retries: Int, timeout: Int): Unit = {
```

Comment:
```suggestion
        def tryFetch(retries: Int, timeout: Duration): Unit = {
```

Consider using `Duration` here and updating the code accordingly.

Reply:
changed